reV/supply_curve/points.py

```diff
@@ -548,6 +578,31 @@ def _check_excl(self):
             )
             raise EmptySupplyCurvePointError(msg)
 
+    def _check_zone_mask(self):
+        """
+        Check that the zone mask is the correct size and shape, and that it
+        contains only values of 0 and 1.
+        """
+        if self._zone_mask is not None:
+            msg = (
+                "Bad zone mask input shape of {} with stated "
+                "resolution of {}".format(
+                    self._zone_mask.shape, self._resolution
+                )
+            )
+            assert len(self._zone_mask.shape) == 2, msg
+            assert self._zone_mask.shape[0] <= self._resolution, msg
+            assert self._zone_mask.shape[1] <= self._resolution, msg
+            assert self._zone_mask.size == len(self._gids), msg
+
+            if not np.isin(self._zone_mask, [0, 1]).all():
+                msg = (
+                    "zone_mask includes unexpected values. All values must be "
+                    "in the domain: [0, 1]"
```

Comment:
This is a bit misleading since zone mask is actually a boolean array that I assume is implicitly cast to ints during the `np.isin` operation (and probably during the `self.zone_mask == 0` line above)

Reply:
Looks good to me!